include/mbgl/gfx/rendering_stats.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+namespace mbgl {
+namespace gfx {
+
+struct RenderingStats {
+    bool isZero() const;
```

Comment:
`isEmpty()` ?

Reply:
I would say that being "empty" is not the right term in this context.